src/app/core/database/database-resolver.service.ts

```diff
@@ -65,14 +60,25 @@ export class DatabaseResolverService {
 
   async initDatabasesForSession(session: SessionInfo) {
```

Comment:
Is `initDatabasesForSession` used only to create/initialize a database for the current web session, such as in demo mode only?

Reply:
```suggestion
  /**
   * Connect the database(s) for the current user's "session",
   * i.e. configuring the access for that account after login
   * (especially for local and remote database modes)
   */
  async initDatabasesForSession(session: SessionInfo) {
```

Reply:
less for demo "mock" mode but rather for local and synced mode, actually. This connects the database instances to the correct persistent database in the browser (and server) for that specific user account.

Does the code comment make that clear now? Or what else should we add as explanation?

Reply:
Thanks, now the code comment makes sense.